### ISSUE TITLE
Issue not caught by compiler

### ISSUE THREAD

**Issue by rob-reynolds** [cqframework/clinical_quality_language#1227]
The following is a bug that was working in a prior version of CQL, but does not work in the version of CQL used by https://releases.smilecdr.com/releases/2023/11/PRE/2023-11-PRE-35/:
```
define function ToString(value id):
  value.value
```

It throws the following error at runtime using the noted CQL version:
```
{
   "resourceType": "OperationOutcome",
   "issue": [
        {
           "severity": "error",
           "code": "processing",
           "diagnostics": "HAPI-0389: Failed to call access method: org.opencds.cqf.cql.engine.exception.CqlException: Could not resolve call to operator 'ToString(java.lang.Class)' in library 'DQMFHIRHelpers'."
        }
    ]
}
```

JP commented: Obviously we should catch that in the compiler.  

Note this can't be reproduced using the current version of the VSCode Extension (v0.6.0) because the current VSCode Extension uses a prior version of CQL in which the above evaluates without error, i.e., doing the above works in the current VSCode Extension.

**Comment by JPercival** [comment]
Potential duplicate of #1223

**Comment by JPercival** [comment]
Duplicate, fixed by #1260
